<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>threeplayers</class>
 <widget class="QDialog" name="threeplayers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1250</width>
    <height>798</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="cartaJug2Num1">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>330</y>
     <width>92</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cartaJug1Num2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>92</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cartaJug2Num2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>330</y>
     <width>92</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cartaJug2Num3">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>330</y>
     <width>92</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cartaJug1Num3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>92</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cartaCasaNum3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>92</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cartaCasaNum2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>92</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cartaCasaNum1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>92</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cartaJug1Num1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>92</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cartaJug3Num3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>530</y>
     <width>92</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cartaJug3Num2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>530</y>
     <width>92</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cartaJug3Num1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>92</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>92</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Rockwell&quot;;
</string>
   </property>
   <property name="text">
    <string>Pedir carta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>640</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Rockwell&quot;;
</string>
   </property>
   <property name="text">
    <string>Detenerse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1250</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1250</width>
     <height>700</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>Jugador Uno</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>Jugador Tres</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>280</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>Jugador Dos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>Casa</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>cartaJug2Num1</zorder>
  <zorder>cartaJug1Num2</zorder>
  <zorder>cartaJug2Num2</zorder>
  <zorder>cartaJug2Num3</zorder>
  <zorder>cartaJug1Num3</zorder>
  <zorder>cartaCasaNum3</zorder>
  <zorder>cartaCasaNum2</zorder>
  <zorder>cartaCasaNum1</zorder>
  <zorder>cartaJug1Num1</zorder>
  <zorder>cartaJug3Num3</zorder>
  <zorder>cartaJug3Num2</zorder>
  <zorder>cartaJug3Num1</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
